build_tools/mako/configuration.py

```diff
@@ -236,10 +236,12 @@ def get_s20_default_target_list(skipped_target=None,
                     skipped_target=["vlk2"],
                     compilation_flags={
                         'cpu': [
-                            "--iree-flow-dispatch-formation-enable-operand-fusion"
+                            "--iree-flow-dispatch-formation-enable-operand-fusion",
+                            "-iree-llvm-loop-unrolling=true"
                         ],
                         'cpu3t': [
-                            "--iree-flow-dispatch-formation-enable-operand-fusion"
+                            "--iree-flow-dispatch-formation-enable-operand-fusion",
```

Comment:
We don't need to enable `operand-fusion` on CPU. We can delete that in a follow up PR